### PR TITLE
Using paged.js to polyfill the CSS spec and improve printing of approved activity reports

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
Looking for some feedback regarding this implementation.

Installed a third party library that polyfills upcoming CSS spec for using the @page query selector and associated psedo-selector. It also (seems to) handle the break-inside, break-after properties in a more predictable way.

It also does a few other things out of the box we may find interesting, like automatically displaying a header as a static header across pages and, with the aforementioned polyfill, we can use the 

```css

@page {
  @bottom-right {
    content: 'Page ' counter(page); 
  }
}

```

syntax that is forthcoming.

Using this library will, in theory, be something we can do until our user's browsers get to the point where printing is handled a bit better in CSS naturally.

## How to test

Right now, the library is used only on the new approved activity report page, and only when you click the big old blue "print to pdf" button.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-244 (Kinda)

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/pages/ApprovedActivityReport/components/ViewTable.css [hunk 1]
```diff
@@ -44,26 +44,24 @@
...
         padding: 0.5em 1em;
         vertical-align: top;
-        width: 250px;
+        width: 200px;
     } 
 
     .ttahub-activity-report-view-table td {
         padding: 0 1em;
-    }
+    }
```
Comment (jasalisbury) [inline frontend/src/pages/ApprovedActivityReport/components/ViewTable.css:65]:
```suggestion
    }
```

**Review by jasalisbury — COMMENTED** [review]
Might just be my report but the resulting print seems worse off with this change:
The context label is on one page with the value on a different page. Actually context is repeated on this report for some reason.
Screenshot from 2021-07-28 09-54-56
Screenshot from 2021-07-28 09-58-20

Also the whole page is updated with the print CSS (not just the print preview) which may be the point of pagedjs, but still surprised me a bit
Screenshot from 2021-07-28 09-55-17

Also I have a random blank page when printing.

**Comment by thewatermethod** [comment]
ah interesting, I have none of those issues. I'll have to poke around a bit more

**Comment by jasalisbury** [comment]
> ah interesting, I have none of those issues. I'll have to poke around a bit more

Let me know if you want to pair and I can show the report live

**Comment by thewatermethod** [comment]
Don't see a way to get consistent results on this

**Pull request closed without merging**